CHANGELOG.md

```diff
@@ -1,6 +1,14 @@
 Changelog
 =========
 
+## TBC
+
+### Fixes
+
+* Fix nil dereference in SQL Active Record `bind` key.
```

Comment:
`:binds` instead of `bind`, although I would probably reword this to something like "Account for missing `:binds` key in `sql.active_record` ActiveSupport notifications"